changelog.txt

```diff
@@ -1,3 +1,75 @@
+1.9.0-beta1 (2018-02-10)
+------------------------------------------------------------------------
+Note: OpenTTD was migrated to GitHub for 1.9, so SVN revision and FlySpray numbers have been replaced with Pull Requests and Issue numbers
+- Feature: Group liveries, and livery window usability enhancements (#7108)
+- Feature: Overhaul of music system, support MPSMIDI music files (TTD DOS/TTO) (#6839)
+- Feature: [Win32] Rewritten DMusic and WinMM music drivers
+- Feature: [Win32] New XAudio2 sound driver
+- Feature: [Linux] New Fluidsynth music driver
+- Feature: Framerate display window (#6822)
+- Feature: [NewGRF] Ability for industries & houses to produce and/or accept up to 16 different cargoes
+- Feature #6610: Allow towns to build houses on road turns (#6758)
+- Feature: Replace independent map scrolling GUI settings with single option, and add choice to not lock cursor position when scrolling (#6756)
+- Feature #6459: API for querying network clients from GS (#6736)
+- Feature: New Russian Ruble & Mexican Peso as currencies (#6678, #7035)
+- Feature #4186: Append '(City)' after city names in the town directory
+- Feature #986: Automatic save when losing connection to a network game
+- Add: [NewGRF] Flag to hide rail types from construction (#7118)
+- Add: [NewGRF] Airport animation trigger for plane landing (#6334)
+- Add #4115: Default starting company colour setting (#6998)
+- Add: Mixer feature for streaming sampled music
+- Add #6460: [AI] start_date parameter for Random AIs on new game
+- Add: BACKSPACE can be mapped as a hotkey
+- Add: 32bpp SSE2 blitter palette animator (#6795)
+- Change: Make ships turn slowly when changing direction and stop when going up or down a lock (#7065, #7150)
+- Change: Improve ship pathfinder performance
+- Change: Decouple GUI timing from game ticks (#6780)
+- Change: Desert tiles are now half-desert if a neighboured tile is non-desert or sea/coast (#4754)
+- Change: Gradually slow down aircraft speed on breakdown (#6932)
+- Change: [NewGRF] Increase rail type & cargo type limits to 64
+- Change: [NewGRF] Increase size of persistent storage to 256
+- Change: [NewGRF] Use last OpenTTD SVN revision in NewGRF version number (#6843)
+- Change: Clone tool in depot window now behaves like clone button in vehicle window (#6754)
+- Change #6397: Keep town growth rate in sync with house count
+- Change: Scale default FreeType font size selection by UI zoom level
+- Change: [OSX] Reversed pinch to zoom behaviour
+- Change: Switch town growth rate and counter to actual game ticks (#6763)
+- Change: Non-tag revisions are now named '<commitdate>-<branch>-g<shorthash><modified>'
+- Change: Rewrite several bits of documentation in markdown
+- Fix: Potential crash during shutdown after unpacking downloaded content (#7185)
+- Fix #6584: Text layout in engine preview dialogue windows was
```

Comment:
Incomplete sentence here: I would just remove "windows was" at the end, so it just reads "Fix #6584: Text layout in engine preview dialogue"